client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/style.scss

```diff
@@ -498,6 +498,8 @@ $design-button-primary-color: rgb( 17, 122, 201 );
 		line-height: 1.25rem;
 		margin: 8px 0;
 		width: 100%;
+		overflow: visible;
+    	margin-bottom: 48px;
```

Comment:
I think this should be added to the “view more“ button or “previews“ wrapper. Also, I think we don't need to set `overflow: visible`. What do you think 🤔

Reply:
The style is applied for `.generated-design-picker__view-more`, also without `overflow: visible` it looks like this
https://d.pr/i/OItbjC

Reply:
[nit] Oh, I think we can merge this with L499.

```css
margin: 8px 0 48px;
```

Reply:
[nit] we can merge this with L499

```css
margin: 8px 0 48px;
```